app/Models/Course.php

```diff
@@ -44,6 +44,11 @@ public function lessons()
         return $this->hasMany(Lesson::class);
     }
 
+    public function getPriceAttribute($price)
+    {
+        return number_format($price);
```

Comment:
$this->price

Reply:
em viết theo ví dụ này trong docs của laravel ạ, có được không a?
![image](https://user-images.githubusercontent.com/77009500/136241128-333bbe66-bb75-4b37-8acb-4700cc6f4de8.png)

Reply:
đổi tên attribute là được em nhé
ví dụ
```
public function getPriceFormatedAttribute
{
    return number_format($this->price)
}
```